lookup-table.yaml

```diff
@@ -587,3 +587,15 @@
     - packageId: vram-guild.canvas
       loaders:
         - fabric
+- id: plasmo_voice
```

Comment:
When adding a mod to the lookup table, you need to order it alphabetically. First by the publisher and then by the modid.
This mod should be between `PepperCode1.continuity` and `QuantumFusion.dashloader`.

Reply:
This is actually my fault, because I never actually started the guide to making a manifest ;)